docker/requirements.txt

```diff
@@ -5,11 +5,10 @@ imageio-ffmpeg
 ipywidgets<9.0.0
 jupyterlab<5.0.0
 meshio
-nest_asyncio
 trame>=2.5.2
-trame-client>=2.12.7
-trame-jupyter-extension>=2.0.1
-trame-server>=2.11.7
-trame-vtk>=2.5.8
-trame-vuetify>=2.3.1
+trame_client>=2.12.7
+trame_jupyter_extension>=2.0.1
```

Comment:
`trame_jupyter_extension>=2.0.1` is the only one with `_`

Reply:
I think pip resolves these either way?

Reply:
I guess I got it from the generated readme... https://pypi.org/project/trame-jupyter-extension/

I should update the readme to match the actual package name from pypi.